pkg/app/master/command/debug/handle_kubernetes_runtime.go

```diff
@@ -136,22 +153,6 @@ func HandleKubernetesRuntime(
 			"ec.count": len(pod.Spec.EphemeralContainers),
 		}).Debug("target pod info")
 
-	if commandParams.ActionListDebuggableContainers {
```

Comment:
@eharris128 it might still be good to have an option to list all debuggable containers in a given pod, but if we have both options (to list them for all pods and for a specific pod) then we'll need a separate flag for this, so you can explicitly select what option you want.

Reply:
Yeah supporting both paths - per pod and across pods seems reasonable to me.

I do not remember how the `--list-debuggable-containers` flag performs right now (or before my change) if one passed in `--pod` as well. 

Feels like adding a new flag to differentiate by one pod versus all pods would be the clearest UX.

Reply:
@eharris128 how about `--list-debuggable-pod-containers`

Reply:
`--list-debuggable-pod-containers` For this flag we would require users to provide a `--pod` as well? 

Making sure I follow.